Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Community Script Hook V.NET")]
+[assembly: AssemblyDescription("An ASI plugin for Grand Theft Auto V, which allows running scripts written in any .NET language in-game.")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("ScriptHookVDotNet")]
+[assembly: AssemblyCopyright("Copyright © 2015 crosire")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+// Setting ComVisible to false makes the types in this assembly not visible
+// to COM components.  If you need to access a type in this assembly from
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+// The following GUID is for the ID of the typelib if this project is exposed to COM
+[assembly: Guid("61165c1a-6bfe-4776-b371-5a3fdb1a2ed7")]
+
+// Version information for an assembly consists of the following four values:
+//
+//      Major Version
+//      Minor Version
+//      Build Number
+//      Revision
+//
+// You can specify all the values or you can default the Build and Revision Numbers
+// by using the '*' as shown below:
+// [assembly: AssemblyVersion("1.0.*")]
+[assembly: AssemblyVersion("3.0.0")]
```

Comment:
This will break automatic versioning via a preprocessor define. Fortunately it looks like there are ways to work around that with a custom MSBuild task: http://www.andrewconnell.com/blog/Using-MSBuild-to-generate-assembly-version-info-at-build-time-including-SubVersion-fix.

Reply:
It'll take quite long time for me to read. Will change this line after testing memory stuff good enough.

Reply:
You don't have to do this. I just noted this here for later. The automated build version wasn't in the V3 branch yet anyway.